pkgs/development/compilers/idris2/default.nix

```diff
@@ -1,22 +1,29 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper
-, clang, chez
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, clang
+, chez
+, gmp
 }:
 
+# NOTICE: An `idris2WithPackages` is available at: https://github.com/claymager/idris2-pkgs
```

Comment:
Interesting... We should probably dig into that and get some inspiration.

Reply:
I am more hopeful that @claymager will upstream at least some of it, haven't checked the license so maybe I could check that and upstream some of the stuff.